controllers/backup/resourcecollector.go

```diff
@@ -470,3 +470,223 @@ func (ctrl *controller) GetServiceAccountSpec(podspec v1.PodSpec, namespace stri
 
 	return nil
 }
+
+func (ctrl *controller) getEnvFromsForPodSpec(podspec v1.PodSpec, namespace string,
```

Comment:
Please handle duplicate codes in the function

Reply:
handled